src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -46,21 +53,129 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(roundEntity);
   }
-  
 
-  private String getAuthToken() {
-    String clientId = "1c29ff191b444611a6d9dbb4a354642f";
-    String clientSecret = "6b92b6eb8fe24d5e98178407d3099bd8";
-    SpotifyApi spotifyApi = new SpotifyApi.Builder().setClientId(clientId).setClientSecret(clientSecret).build();
-    ClientCredentialsRequest clientCredentialsRequest = spotifyApi.clientCredentials().build();
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String playlistUrl = getParameter(request, "playlist-link", "");
+    setvideoId(playlistUrl);
+    response.sendRedirect("/game.html");
+  }
 
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
```

Comment:
Great first pass at this javadoc. Note that according to the style guide, you should replace "@return" with just "Returns", so this doc would read:
"Returns the request parameter, or the default value if the parameter was not specified by the client."

See:
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

The point is that all javadocs should start with a summary fragment before any @returns or @param blocks. In general I find that a summary fragment and possibly an additional sentence or two is enough, and I usually don't need to use @return or @param blocks.